src/cache/lru_cache_manager/lru_cache_manager.h

```diff
@@ -6,7 +6,7 @@
 
 #ifndef LRU_CACHE_MANAGER_H
 #define LRU_CACHE_MANAGER_H
-#include "../result_set_cache_includes.h"
+#include "../cache_includes.h"
 #include "./lru_queue.h"
 
 /**
```

Comment:
if LRUCacheManager simply wraps LRUQueue, consider removing it or typedef LRUCacheManager as a pointer of LRUQueue.

Reply:
done